lib/dispatch-first-click.js

```diff
@@ -0,0 +1,24 @@
+var tagsToFocus =
+  '["text", "textarea", "date", "datetime-local", "email", "number", "month", "password", "search", "tel", "time", "url", "week" ]'
+
+module.exports = function(webView, event) {
+  var point = webView.convertPoint_fromView(event.locationInWindow(), null)
+  var x = point.x
+  var y = webView.frame().size.height - point.y // the coord start from the bottom instead of the top
+  return (
+    'var el = document.elementFromPoint(' + // get the DOM element that match the event
```

Comment:
I haven't had a chance to submit a fix but this throws an error when the click target is the window title bar, since there are no elements to click in that region.